packages/smart-contracts/deploy/utils-zk.ts

```diff
@@ -0,0 +1,172 @@
+import { Provider, Wallet, Contract } from 'zksync-web3';
+import * as hre from 'hardhat';
+import { Deployer } from '@matterlabs/hardhat-zksync-deploy';
+import { formatEther } from 'ethers/lib/utils';
+import { BigNumberish } from 'ethers';
+
+import { config } from 'dotenv';
+import { networkRpcs } from '@requestnetwork/utils/dist/providers';
+
+config();
+
+const accounts = process.env.DEPLOYMENT_PRIVATE_KEY
```

Comment:
Seems weird, no? Don't we just have 1 PK for deployment/

Reply:
I just copied the logic from hardhat.config.ts file, we get the accounts the same way

Reply:
I think it can be simplified with the only key you need for deployment.

For example the `DEPLOYER_MASTER_KEY` is never used in this context. 
I think you can also remove DEPLOYMENT_PRIVATE_KEY, and only use ADMIN_PRIVATE_KEY. Both of them exists in the original hardhat config, as we needed a distinction when we deployed the contract with the scripts not using `CREATE2`.